src/main/java/base/DriverManager.java

```diff
@@ -0,0 +1,71 @@
+package base;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+
+public class DriverManager {
+    public static WebDriver driver;
+
+    public static void initDriver() {
+        String browserType = Config.getProperty("browser").toLowerCase();
+        if (browserType.equals("chrome")) {
+            WebDriverManager.chromedriver().setup();
+            DriverManager.driver = new ChromeDriver();
+        } else if (browserType.equals("firefox")) {
+            WebDriverManager.firefoxdriver().setup();
+            DriverManager.driver = new FirefoxDriver();
+        } else {
+            throw new IllegalArgumentException("Unsupported browser type: " + browserType);
+        }
+
+        DriverManager.driver.manage().window().maximize();
+
+
+    }
+
+    public static void navigateToRailWay() {
+        String railwayUrl = Config.getProperty("railway.url");
+        DriverManager.driver.get(railwayUrl);
+        // railway = WebDriverConfig.driver.getWindowHandle();
+    }
+
+    public static void navigateToMailPage() {
+        String mailUrl = Config.getProperty("tempmail.url");
+        DriverManager.driver.get(mailUrl);
+        //email = WebDriverConfig.driver.getWindowHandle();
+    }
+    
+
+    public static void waitForElementToBeVisible(By locator, int timeoutInSeconds) {
+        WebDriverWait wait = new WebDriverWait(DriverManager.driver, Duration.ofSeconds(timeoutInSeconds));
+        wait.until(ExpectedConditions.visibilityOfElementLocated(locator));
+    }
+
+    public static void waitForClickableElement(String xpathExpression) {
```

Comment:
dynamic timeout variable by passing param instead of fixing timeout value. like waitForElementToBeVisible